executor/infoschema_reader.go

```diff
@@ -117,6 +117,8 @@ func (e *memtableRetriever) retrieve(ctx context.Context, sctx sessionctx.Contex
 			err = e.setDataFromSessionVar(sctx)
 		case infoschema.TableTiDBServersInfo:
 			err = e.setDataForServersInfo()
+		case infoschema.TableTiFlashReplica:
```

Comment:
Need to add this check in `buildMemTable` function too.

Reply:
done.